scripts/battle/GameState.cs

```diff
@@ -99,10 +100,21 @@ public class GameState {
             ProcessCombo(matchingCombo);
         } else { ComboStackChangedCustomEvent?.Invoke(this, EventArgs.Empty); }
         DeselectDeadEnemy();
+        HideDeadEnemies();
```

Comment:
Move to the end of `PlaySelectedCard()`, after `Hand.DiscardCard()`

Reply:
Done